src/rootPages/Designer/properties/views/ABViewMenu.js

```diff
@@ -0,0 +1,795 @@
+                  },
+               ],
+               on: {
+                  onChange: () => {
+                     this.onChange();
+                  },
+               },
+            },
+            {
+               id: ids.menuAlignment,
+               name: "menuAlignment",
+               view: "richselect",
+               label: L("Menu Alignment"),
+               value: ABViewMenuPropertyComponentDefaults.menuAlignment,
+               labelWidth: uiConfig.labelWidthXLarge,
+               options: [
+                  {
+                     id: "ab-menu-left",
+                     value: L("Left"),
+                  },
+                  {
+                     id: "ab-menu-center",
+                     value: L("Center"),
+                  },
+                  {
+                     id: "ab-menu-right",
+                     value: L("Right"),
+                  },
+               ],
+               on: {
+                  onChange: () => {
+                     this.onChange();
+                  },
+               },
+            },
+            {
+               id: ids.menuInToolbar,
+               name: "menuInToolbar",
+               view: "checkbox",
+               labelRight: L("Put menu in toolbar"),
+               value: ABViewMenuPropertyComponentDefaults.menuInToolbar,
+               labelWidth: uiConfig.labelWidthCheckbox,
+               on: {
+                  onChange: () => {
+                     this.onChange();
+                  },
+               },
+            },
+            {
+               name: "toolbarFieldset",
+               view: "fieldset",
+               label: L("Toolbar Settings:"),
+               labelWidth: uiConfig.labelWidthLarge,
+               body: {
+                  view: "layout",
+                  type: "clean",
+                  padding: 10,
+                  rows: [
+                     {
+                        id: ids.menuPadding,
+                        name: "menuPadding",
+                        view: "counter",
+                        label: L("Toolbar padding"),
+                        value: ABViewMenuPropertyComponentDefaults.menuPadding,
+                        labelWidth: uiConfig.labelWidthLarge,
+                        on: {
+                           onChange: () => {
+                              this.onChange();
+                           },
+                        },
+                     },
+                     {
+                        id: ids.menuTheme,
+                        name: "menuTheme",
+                        view: "richselect",
+                        label: L("Toolbar theme"),
+                        value: ABViewMenuPropertyComponentDefaults.menuTheme,
+                        labelWidth: uiConfig.labelWidthLarge,
+                        options: [
+                           {
+                              id: "white",
+                              value: L("White (Default)"),
```

Comment:
Really small, but can you change this value to "White". It bugged me in v1 because gray is actually our default...